src/v2/view-builder/views/consent/ConsentViewHeader.js

```diff
@@ -0,0 +1,55 @@
+import { View, _ } from 'okta';
+import hbs from 'handlebars-inline-precompile';
+
+const defaultLogo = '/img/logos/default.png';
+
+const ConsentViewHeader = View.extend({
+  className: 'consent-title detail-row',
+  template: hbs`
+    {{#if clientURI}}
+    <a href="{{clientURI}}" class="client-logo-link" target="_blank">
+    {{/if}}
+    {{#if customLogo}}
+      <img class="client-logo custom-logo" src="{{customLogo}}" alt="aria logo" aria-hidden="true" />
+    {{else}}
+      <img class="client-logo default-logo" src="{{defaultLogo}}" alt="aria logo" aria-hidden="true" />
+    {{/if}}
+    {{#if clientURI}}
+      </a>
+    {{/if}}
+    <h1>
+      <span class="title-text">
+        {{#if isAdminConsent}}
```

Comment:
hmm.. why such difference? did you check with UX to see if we can use same text?

Reply:
I am changing it to the text used in the mock which didn't carryover in v1 development: https://oktawiki.atlassian.net/wiki/spaces/PM/pages/1074627985/Spec+End+to+End+Identity+Attestation#User-Flow-Diagram

We shouldn't re-use the text for Enduser because it's grammatically incorrect.
For example, if the text is not updated it would read:
```
App wants to access
- View your phone number
```
vs 
```
App wants to
- View your phone number
```
For admin it works because 
```
App wants to access
- Resource and Policies
```

Reply:
I'll confirm with a designer though.

Reply:
Confirmed with @ZoeMonosson-okta that the header texts should be different

Reply:
It's not ideal to have diverged logic here.
Ideally, View Factory is only place doing dispatching (find the right view for particular remediation form)
If those views share some logic, extra a base view and extends from it.